client/src/js/base/__tests__/Icon.test.js

```diff
@@ -1,5 +1,6 @@
 import { Tooltip } from "../Tooltip";
 import { Icon } from "../Icon";
+import { StyledIcon } from "../Icon";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['../Icon' import is duplicated.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5219009)